manual/manual/cmds/unified-options.etex

```diff
@@ -386,6 +386,32 @@ command line.}
 }%notop
 
 \notop{%
+\item["-opaque"]
+When the native compiler compiles a module, by default it produces
```

Comment:
I think we should take more care here about "module" versus "compilation unit", as above.